common/service/dynamicconfig/constants.go

```diff
@@ -73,7 +72,6 @@ var keys = map[Key]string{
 	EnableFailoverManager:               "system.enableFailoverManager",
 	EnableStickyQuery:                   "system.enableStickyQuery",
 	EnablePriorityTaskProcessor:         "system.enablePriorityTaskProcessor",
-	EnableAuthorization:                 "system.enableAuthorization",
```

Comment:
Please revert this one, it's used by our internal server.

Reply:
Reverted and put them all in a same code block for https://github.com/uber/cadence/issues/3861 later